app/controllers/api/comments_controller.rb

```diff
@@ -9,7 +9,8 @@ def create
   end
 
   def index
-    render json: current_debate.retrieve_comments.map { |comment| CommentSerializer.new(comment) }
+    render json: { debate_closed: current_debate.closed?,
```

Comment:
Why is this needed here? Maybe we should add a debate resource here instead of only `debate_closed` flag?